pilot/pkg/model/push_context.go

```diff
@@ -1441,14 +1458,27 @@ func (ps *PushContext) SetDestinationRules(configs []config.Config) {
 	namespaceLocalDestRules := make(map[string]*processedDestRules)
 	exportedDestRulesByNamespace := make(map[string]*processedDestRules)
 	rootNamespaceLocalDestRules := newProcessedDestRules()
+	inheritedConfigs := make(map[string]*config.Config)
 
 	for i := range configs {
 		rule := configs[i].Spec.(*networking.DestinationRule)
+
+		if features.EnableDestinationRuleInheritance && rule.Host == "" {
+			if t, ok := inheritedConfigs[configs[i].Namespace]; ok {
+				log.Warnf(
+					"Namespace/mesh-level DestinationRule is already defined for %q at time %v. Ignore %q which was created at time %v",
```

Comment:
nit: probably more useful to show the name of the existing one rather than timestamp?

Can we add an analyzer (can be followup)?

Reply:
analyzer issue: https://github.com/istio/istio/issues/30626